files-csharp/FastestWayToReadATextFileInCsharp/Tests/WaysToReadATextFileInCsharpTests.cs

```diff
@@ -1,81 +1,94 @@
-﻿using FastestWayToReadATextFileInCsharp;
+﻿namespace Tests;
 
-namespace Tests
+public class WaysToReadATextFileInCsharpTests
 {
-    public class WaysToReadATextFileInCsharpTests
-    {
-        private static readonly string _sampleFilePath
-            = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName, "SampleTextFile.txt");
-
-        private readonly WaysToReadATextFileInCsharp _textFileReader;
+    private readonly WaysToReadATextFileInCsharp _textFileReader;
 
-        public WaysToReadATextFileInCsharpTests()
-        {
-            _textFileReader = new WaysToReadATextFileInCsharp(_sampleFilePath);
-        }
-
-        private readonly string _expected
-            = """
+    private const string ExpectedText
+        = """
             Integer facilisis ex libero, ut suscipit leo blandit non. Vivamus nec ipsum orci.
             Proin nec mauris dui. Proin at felis et eros commodo aliquet.
             Pellentesque lacinia porta leo, non accumsan turpis sagittis et.
             Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere.
             """;
+    
+    private static readonly string _tempFilePath = Path.GetTempFileName();
```

Comment:
It is good practice to clean up after ourselves, so we should try to ensure we remove this temp file when the test is done, so we can just remove the static here and implement IDisposable to delete the file.

Reply:
Please also fix the naming convention of the _tempFilePath to be of a static readonly string variable.